frontend/src/toolbar/index.tsx

```diff
@@ -30,3 +30,13 @@ initKea()
         container
     )
 }
+
+// Expose `window.getToolbarReduxState()` to make snapshots to storybook easy
+if (typeof window !== 'undefined') {
+    // Disabled in production to prevent leaking secret data, personal API keys, etc
+    if (process.env.NODE_ENV === 'development') {
+        ;(window as any).getToolbarReduxState = () => getContext().store.getState()
+    } else {
+        ;(window as any).getToolbarReduxState = () => 'Disabled outside development!'
+    }
```

Comment:
I don't think we need this `else` branch for anything?

Reply:
Yeah, probably not. This is all a stopgap anyway. Soon we'll be taking the snapshots from within storybook itself 👀 📸